src/integrationTest/java/org/opentripplanner/IntegrationTest.java

```diff
@@ -0,0 +1,59 @@
+package org.opentripplanner;
+
+import com.fasterxml.jackson.jaxrs.json.JacksonJsonProvider;
+import org.glassfish.jersey.client.ClientConfig;
+import org.glassfish.jersey.test.JerseyTest;
+import org.glassfish.jersey.test.TestProperties;
+import org.opentripplanner.api.model.JSONObjectMapperProvider;
+import org.opentripplanner.routing.impl.GraphScanner;
+import org.opentripplanner.routing.services.GraphService;
+import org.opentripplanner.standalone.CommandLineParameters;
+import org.opentripplanner.standalone.OTPApplication;
+import org.opentripplanner.standalone.OTPMain;
+import org.opentripplanner.standalone.OTPServer;
+
+import javax.ws.rs.core.Application;
+
+public abstract class IntegrationTest extends JerseyTest {
+
+    private final static int port = 9111;
+
+    private final static String[] args = new String[]{
+            "--basePath",
+            "./src/integrationTest/resources/",
+            "--inMemory",
+            "--router",
+            "bydgoszcz",
+            "--port",
+            Integer.toString(port),
+    };
+
+
+    @Override
+    public Application configure() {
+        set(TestProperties.CONTAINER_PORT, port);
+        System.setProperty("sharedVehiclesApi", "http://ns3114244.ip-54-38-192.eu:8888/get_standard_vehicle_positions"); // mocked database
```

Comment:
To jest publiczne repo, nie wiem czy to dobry pomysł wstawiać tutaj nasz własny URL

Reply:
można, ale docelowo fajnie by było postawić wiremocka czy coś

Reply:
Bez tego odpalanie pojedyńczych tesów z IDE będzie raczej karkołomne

Reply:
to prawda, wiremock byłby idealny